cfgov/jinja2/v1/owning-a-home/_templates/form-explainer.html

```diff
@@ -0,0 +1,235 @@
+
+{# ==========================================================================
+
+   render( data )
+
+   ==========================================================================
+
+   Description:
+
+   Renders the entire form explainer.
+
+   data:               The data structure is as follows:
+                       [
+                         { 'img': 'url-to-form-img'
+                           'terms': [ ... ]
+                         }
+                       ]
+
+   ========================================================================== #}
+
+{% macro render( data ) %}
+<div class="explain" aria-controls="form-explainer">
```

Comment:
This is referencing https://github.com/cfpb/cfgov-refresh/pull/3773/files#diff-3d9dc87803dad1af3ead8842353b8a87R31, is that right?

Reply:
That is correct.